lib/src/component_declaration/component_base.dart

```diff
@@ -579,8 +587,8 @@ abstract class UiProps extends MapBase
   /// Returns a new component with this builder's [props] and the specified [children].
   ReactElement build([dynamic children]) {
     assert(_validateChildren(children));
+    _sharedAsserts();
 
-    _assertComponentFactoryIsNotNull();
```

Comment:
Does this not go in shared asserts too? or is it a null safety thing that we know factory isn't null?

Reply:
Oh it actually did; the diff is a little wonky, but `_assertComponentFactoryIsNotNull` got removed and its contents were inlined into `_sharedAsserts`.

Reply:
ohhh I see! That makes sense! Thanks!